### PR TITLE
Move test out of run_all_in_graph_and_eager_mode in utils_test

### PR DESCRIPTION
**Pull request by autoih**
cc @gabrieldemarmiesse

### CODE REVIEW

File: tensorflow_addons/image/utils_test.py [hunk 1]
```diff
@@ -17,12 +17,41 @@
...
+    )
+    for shape in (2, 4), (2, 4, 1), (1, 2, 4, 1):
+        exp = tf.ones(shape=(1, 2, 4, 1))
+        res = fn(tf.ones(shape=shape))
+        np.testing.assert_equal(exp.numpy(), res.numpy())
+
+
+def test_to_4D_image_with_invalid_shape():
+    errors = (ValueError, tf.errors.InvalidArgumentError)
+    with np.testing.assert_raises_regex(errors, "`image` must be 2/3/4D tensor"):
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/image/utils_test.py]:
```suggestion
    with pytest.raises(errors, match="`image` must be 2/3/4D tensor"):
```

File: tensorflow_addons/image/utils_test.py [hunk 2]
```diff
@@ -17,12 +17,41 @@
...
+        img_utils.to_4D_image(tf.ones(shape=(1,)))
+
+    with np.testing.assert_raises_regex(errors, "`image` must be 2/3/4D tensor"):
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/image/utils_test.py]: Same here

File: tensorflow_addons/image/utils_test.py [hunk 3]
```diff
@@ -17,12 +17,41 @@
...
+        res = fn(tf.ones(shape=shape))
+        np.testing.assert_equal(exp.numpy(), res.numpy())
+
+
+def test_to_4D_image_with_invalid_shape():
+    errors = (ValueError, tf.errors.InvalidArgumentError)
+    with pytest.raises(errors, match="`image` must be 2/3/4D tensor"):
+        img_utils.to_4D_image(tf.ones(shape=(1,)))
+
+    with np.testing.assert_raises_regex(errors, "`image` must be 2/3/4D tensor"):
```
Comment (autoih) [inline tensorflow_addons/image/utils_test.py]:
```suggestion
    with pytest.raises(errors, "`image` must be 2/3/4D tensor"):
```

File: tensorflow_addons/image/utils_test.py (hunk 2 above)
Comment (autoih) [reply tensorflow_addons/image/utils_test.py]: Thanks @gabrieldemarmiesse. Updated.

**Pull request merged**